components/Typography/examples/TypographyMaterialScalableStylesViewController.m

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "TypographyMaterialScalableStylesViewController.h"
+
+#import "MaterialTypography.h"
+#import "MaterialTypographyScheme.h"
+
+@implementation TypographyMaterialScalableStyleViewController {
+  NSArray<NSString *> *_strings;
+  NSArray<NSString *> *_styleNames;
+  NSArray<UIFont *> *_styleFonts;
+  MDCTypographyScheme *_typography;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
+
+  self.tableView.rowHeight = UITableViewAutomaticDimension;
+  self.tableView.estimatedRowHeight = 50.0;
+
+  _strings = @[
+    @"Material Design Components", @"A quick brown fox jumped over the lazy dog.",
+    @"ABCDEFGHIJKLMNOPQRSTUVWXYZ", @"abcdefghijklmnopqrstuvwxyz", @"1234567890",
+    @"!@#$%^&*()-=_+[]\\;',./<>?:\""
+  ];
+
+  _styleNames = @[
+    // Material Font Styles
+    @"Body1",     @"Body1 (Scalable)",     @"Body2",     @"Body2 (Scalable)",
+    @"Caption",   @"Caption (Scalable)",   @"Button",    @"Button (Scalable)",
+    @"Overline",  @"Overline (Scalable)",  @"Subtitle1", @"Subtitle1 (Scalable)",
+    @"Subtitle2", @"Subtitle2 (Scalable)", @"Headline1", @"Headline1 (Scalable)",
+    @"Headline2", @"Headline2 (Scalable)", @"Headline3", @"Headline3 (Scalable)",
+    @"Headline4", @"Headline4 (Scalable)", @"Headline5", @"Headline5 (Scalable)",
+    @"Headline6", @"Headline6 (Scalable)",
+  ];
+
+  _typography =
+      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
+
+  _styleFonts = @[
+    [_typography.body1 mdc_scaledFontAtDefaultSize],
```

Comment:
If I understand correctly, the call to `mdc_scaledFontAtDefaultSize` is unnecessary since the scheme will return *unscaled but scalable* fonts.  Please remove it or add a comment to describe why it's required for clients.

Reply:
Removed.